jenkins/coldlaunch.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash -x
+
+# receive list of apps to run coldlaunch test against
+# i.e. 'calendar clock dialer email'
+APP_LIST=$1
+
+post_to_treeherder() {
+  RAPTOR_BUILD_STATE=$1
+  RAPTOR_APP_NAME=$2
+  TEST_START_TIME=$3
+  TEST_FINISH_TIME=$4
+  TEST_FAILURE=$5
+
+  python --version
+  source raptor-env/bin/activate
+  python --version
+
+  cd post-to-treeherder
+  ./submit-to-treeherder.py \
+    --repository b2g-inbound \
+    --build-state ${RAPTOR_BUILD_STATE} \
+    --treeherder-url https://treeherder.allizom.org/ \
+    --treeherder-client-id raptor \
+    --treeherder-secret ${RAPTOR_TREEHERDER_SECRET} \
+    --test-type cold-launch \
+    --app-name ${RAPTOR_APP_NAME} \
+    --start-time ${TEST_START_TIME} \
+    --finish-time ${TEST_FINISH_TIME} \
+    --test-failure ${TEST_FAILURE}
+}
+
+run_coldlaunch_test() {
+  APP_ORIGIN=$1
```

Comment:
Ah looks like I need to declare them like:

typeset APP_ORIGIN=$1

then that makes them local to that function